ui/apps/platform/src/Containers/NetworkGraph/components/StyleNode.tsx

```diff
@@ -105,32 +138,28 @@ const renderDecorators = (
         y: number;
     }
 ): React.ReactNode => {
-    if (!data.showDecorators) {
-        return null;
-    }
-    const nodeStatus = element.getNodeStatus();
+    const { showPolicyState, networkPolicyState } = data;
     return (
         <>
-            {!nodeStatus || nodeStatus === NodeStatus.default
-                ? renderDecorator(
-                      element,
-                      TopologyQuadrant.upperLeft,
-                      <FolderOpenIcon />,
-                      getShapeDecoratorCenter
-                  )
-                : null}
-            {renderDecorator(
+            {/* {renderDecorator(
                 element,
-                TopologyQuadrant.upperRight,
-                <BlueprintIcon />,
+                TopologyQuadrant.upperLeft,
+                <FolderOpenIcon />,
                 getShapeDecoratorCenter
-            )}
-            {renderDecorator(
+            )} */}
+            {/* {renderDecorator(
                 element,
-                TopologyQuadrant.lowerLeft,
-                <PauseCircle />,
+                TopologyQuadrant.upperRight,
+                <PficonNetworkRangeIcon />,
                 getShapeDecoratorCenter
-            )}
+            )} */}
```

Comment:
I think this comment goes with commented import above.

Reply:
same here, the other branch is branched off of this one so i left them commented for now